index.html

```diff
@@ -8,26 +8,176 @@
     <title>Linus Shaddai Portfolio</title>
   </head>
   <body>
-    <header>
-      <nav>
-        <a href="#">
-          <img src="./img/Logo Placeholder.svg" alt="logo" />
-        </a>
-        <img class="hamburger-menu" src="./img/Union.svg" alt="hambuger menu" />
-      </nav>
-    </header>
+    <section class="head-section">
+      <header>
+        <nav>
+          <a href="#">
+            <img src="./img/Logo Placeholder.svg" alt="logo" />
+          </a>
+          <img
+            class="hamburger-menu"
+            src="./img/Union.svg"
+            alt="hambuger menu"
+          />
+        </nav>
+      </header>
 
-    <section class="headline">
-      <div class="headline-content">
-        <h1>I'm Shaddai,<br />Glad to see you</h1>
+      <section class="headline">
+        <div class="headline-content container">
+          <div class="healine-content-container">
+            <h1>I'm Shaddai,<br />Glad to see you</h1>
+            <p>
+              I’m a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+            <a href="#" class="connect">Lets Connect</a>
+            <ul class="headline-socials">
+              <li>
+                <a href="#"
+                  ><img src="./img/Vector.svg" alt="twitter logo"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="./img/Linkedin icon.svg" alt="Linkedin logo"
+                /></a>
+              </li>
+              <li>
+                <a href="#"><img src="./img/Vector (1).svg" alt="logo" /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="./img/Vector (2).svg" alt="github logo"
+                /></a>
+              </li>
+              <li>
+                <a href="#"><img src="./img/Vector (3).svg" alt="logo" /></a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </section>
+    </section>
+
+    <section class="works-section">
```

Comment:
Please consider changing the value of the CSS `display` property for this main container from `display: block` to `display:grid` as mentioned in the project requirements. please see the screenshot below :point_down:  for more details on this.

![review from 2022-09-22 21-11-30](https://user-images.githubusercontent.com/16392046/191824442-7d76b1fa-7f6e-4f85-9b73-7bd4d3d56f8e.png)

Reply:
Hi @Linushaddai99 kindly note that as per your current implementation of the design the `.work-section` is the container for all the individual projects yet it doesn't use the `display: grid` property it uses the default user-agent style sheet style of `display: block` that's why I recommended this change.